pkg/nodeagent/node_agent.go

```diff
@@ -89,3 +113,31 @@ func GetPodSpec(ctx context.Context, kubeClient kubernetes.Interface, namespace
 
 	return &ds.Spec.Template.Spec, nil
 }
+
+func GetConfigs(ctx context.Context, namespace string, kubeClient kubernetes.Interface) (*Configs, error) {
```

Comment:
We should also update the website to document how to use this configmap

Reply:
Yes, I am planning to create a node-agent configuration document later.